frontend/src/app/components/questions/question-delete/question-delete.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { CrudService } from '../../../services/crud.service';
+import { Router, ActivatedRoute } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
+import { AuthService } from '../../../services/auth.service';
+import { ErrorHandlerService } from '../../../services/error-handler.service';
+import { Question } from 'src/app/models/question';
+
+@Component({
+  selector: 'app-question-delete',
+  templateUrl: './question-delete.component.html',
+  styleUrls: ['./question-delete.component.css']
+})
+export class QuestionDeleteComponent implements OnInit {
+
+  constructor(
+    private errorHandler: ErrorHandlerService,
+    private crud: CrudService,
+    private router: Router,
+    private auth: AuthService,
+    private route: ActivatedRoute) { }
+
+  ngOnInit() {
+    const id = parseInt(this.route.snapshot.paramMap.get('id'), 10);
+
+    console.log('Deleting');
+    this.crud.delete(this.crud.models.QUESTION, id)
+    .subscribe(
+      (res: Response) => {
+        this.errorHandler.showInformativeMessage('Successfully deleted question.');
```

Comment:
Should this component do something else after deleting the question (e.g. redirecting to another component)?